Create common-sonar-L04xMTW-GL04xMT.rst

AlksSAV opened the pull request.

AlksSAV commented:
Added support for new sonars. More information on the [discuss.ardupilot.](https://discuss.ardupilot.org/t/new-type-sonar-with-nmea-out/119111)

Hwurzburg commented on common/source/docs/common-sonar-L04xMTW-GL04xMT.rst:
```diff
@@ -0,0 +1,42 @@
+--------------------------
+.. image:: ../../../images/Sonars nmea.png
+
+The L04xMTW / GL04xMT provides distance measurements using the NMEA protocol over serial at 115200 baud.
```
appears that the sonar data is converted to NMEA by the Nano...so this is a bit misleading

AlksSAV replied:
yes, the sonar data is converted to the nmea data stream.  An arduino nano is used to convert the data

Hwurzburg commented on common/source/docs/common-sonar-L04xMTW-GL04xMT.rst:
```diff
@@ -0,0 +1,42 @@
+
+Connecting and Configuring
+--------------------------
+.. image:: ../../../images/Sonars nmea.png
```
cant have whitespace in filename....also document not in a TOCtree....needs addition on rangefinder landing page TOCtree

rmackay9 commented:
Hi, thanks for this.  I think it would be easier for users if the driver was written in Lua avoiding the need for the external arduino board but in any case these instructions allow users (with some effort) to use previously unusable underwater sonar

AlksSAV commented:
@rmackay9! If I succeed, I will try to do it in a LUA script. 
@Hwurzburg I changed the name of the image and added it to the list. Could you squash the commit I've tried, I'm having trouble with something.

Hwurzburg commented:
you probably needed to rebase on master before trying to squash the 8 commits....I've squashed and pushed back....and will merge once CI passes again (it does locally)
